### ISSUE TITLE
Documentation: All links should open in a new tab

### ISSUE THREAD

**Issue by BartuCulha** [PrimeDAO/prime-deals-dapp#961]
1. navigate here: https://prime-deals-staging.vercel.app/documentation/GettingStarted
2. Click on any hyperlink

### Result:
![Getting-Started-Documentation-Documentation-Prime-Deals (1)](https://user-images.githubusercontent.com/24368665/167135945-79d4a0b5-0645-4af6-9817-5defd742dd5c.png)

Links opens in the same window

### Expected:
Link to open in a new tab

**Comment by dkent600** [comment]
@ogsamovar I think, since some links maybe desired to be in the documentation, and others external, you should code the external anchor tags yourself, by adding  target="_blank" rel="noopener noreferrer"

**Comment by ogsamovar** [comment]
I think this time I can do that by adding HTML links instead of markdown links ~ it should works. Let me try.  In v2 would be nice to have this as a global rule when every external link is automatically opened in a new window.

**Comment by ogsamovar** [comment]
Yes I will manually add those rules to relevant links for this time.

**Comment by ogsamovar** [comment]
It seems that the `target="_blank"` attribute gets ignored / bypassed by the app.

It's not even present when inspecting the code O_o

you can verify it here  https://prime-deals-staging.vercel.app/documentation by clicking and inspecting any external link (like the blogs or the Gnosis Safe website link).

and by checking the raw md file that contains that rule https://raw.githubusercontent.com/PrimeDAO/deals-docs/main/published-md-files/getting-started.md


I grabbed a video to show you that the link are being opened in the same window.

https://user-images.githubusercontent.com/99011533/167849374-844b190e-f8d1-4a91-9e6d-195a019be4f2.mp4
